crates/oxc_minifier/src/ast_passes/peephole_replace_known_methods.rs

```diff
@@ -549,7 +533,8 @@ mod test {
         fold_same("x = 'abcde'.charCodeAt(5)"); // or x = (0/0)
         fold_same("x = 'abcde'.charCodeAt(-1)"); // or x = (0/0)
         fold_same("x = 'abcde'.charCodeAt(y)");
-        fold("x = 'abcde'.charCodeAt()", "x = 97");
+        // Seems that it does not handle this case
+        // fold("x = 'abcde'.charCodeAt()", "x = 97");
```

Comment:
yeah i tried and_then, but it means you can't handle cases such as this

Reply:
Yeah. Actually closure compiler doesn't handle this right now in the deprecated REPL. I'll take an investigation on the weekend to determine which approach should we use.

Reply:
Actually it does not handle it:
```
(base) ... ~ % google-closure-compiler
The compiler is waiting for input via stdin.
x = 'abcde'.charCodeAt()
x="abcde".charCodeAt();
```